src/pages/CertificatesViewAll.js

```diff
@@ -21,8 +21,58 @@ const headersMap = {
     'Carbon Embodiment',
     'Status',
   ],
-  emma: ['Date', 'Commitment'],
-  reginald: [],
+  emma: [
+    'Date',
+    'H2 Batch size',
+    'H2 Certificate holder',
+    'Carbon Embodiment',
+    'Status',
+  ],
+  reginald: [
+    'Date',
+    'H2 Batch size',
+    'H2 Certificate holder',
+    'Carbon Embodiment',
+    'Status',
+  ],
+}
```

Comment:
I think you are planning to change this in one of your next PRs , if I am not mistaking , right ? And by change I am referring to add a line or a couple about Connor , right ? ( to get rid of the TypeError: {(intermediate value)(intermediate value) error )

Reply:
will include Connor [key] in the next PR for the reasons in your comment